### DISCUSSION TITLE
/sb/v4/* endpoint not working for europe

### DISCUSSION THREAD

**Discussion by Luscha** [amzn/ads-advanced-tools-docs#140] [Q&A]
I'm trying to call the POST `/sb/v4/ads/list` endpoint as per [documentation ](https://advertising.amazon.com/API/docs/en-us/sponsored-brands/3-0/openapi/prod#tag/Ads/operation/ListSponsoredBrandsAds).

I get back a 404 telling me that no resources are found.

![image](https://github.com/amzn/ads-advanced-tools-docs/assets/24476025/e60a9c22-8231-4af2-95f2-7410793b1664)

I tried with several endpoints of both sb V4 and V3, and this problem seems to affect only endpoints `sb/v4/*`.

Is there any trouble with the European sb v4 endpoints? I couldn't find anything in the documentation

**Comment by elizafarley** [comment]
Hi @Luscha,

I'm not aware of any issue that might be impacting the sb/v4/* endpoints in the EU, but if you continue to experience this I'd recommend reaching out to [tech support in Jira](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2) so the engineering team can take a look.
